authors/models.py

```diff
@@ -22,13 +23,11 @@ class Author(models.Model):
     # followers: Authors, added by related name, see AuthorFollowingRelation
 
     def is_internal(self):
-        # true if id is a uuid, else (a url) it's external
-        # https://stackoverflow.com/a/33245493
         try:
-            _ = uuid.UUID(self.id)
-            return True
-        except ValueError:
+            _ = Request('GET', self.id).prepare()
```

Comment:
This `prepare()` call verifies the parameter has valid URL scheme, which should do the job for us. No actual request is sent.